operator/option/config.go

```diff
@@ -335,7 +339,8 @@ func (c *OperatorConfig) Populate() {
 	// AWS options
 
 	c.AWSReleaseExcessIPs = viper.GetBool(AWSReleaseExcessIPs)
-	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPI)
+	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPI) ||
+		viper.GetBool(UpdateEC2AdapterLimitViaAPIv2)
```

Comment:
Given the above change, this would then be:
```suggestion
	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPIDeprecated) ||
		viper.GetBool(UpdateEC2AdapterLimitViaAPI)
```